src/arch/arm/pmu.cc

```diff
@@ -543,8 +543,9 @@ PMU::CounterState::detach()
         sourceEvent->detachEvent(this);
         sourceEvent = nullptr;
     } else {
-        debugCounter("detaching event not currently attached"
-            " to any event\n");
+      DPRINTFS(PMUVerbose, &pmu,
+          "detaching event %d not currently attached to any event [counterId "
+          "= %d, sourceEvent = %#x]\n", eventId, counterId, sourceEvent);
```

Comment:
Not a big deal, but why are you printing sourceEvent if it is equal to nullptr?

Reply:
Oh yeah that does not make any sense - I just copied the previous format without thinking about that. Will fix.

Reply:
@arichardson can you push this fix? Thanks.

Reply:
sorry for the delay, fixed now.